packages/styles/_variables.scss

```diff
@@ -46,6 +46,12 @@ $colors: (
   black-25: rgba(0, 0, 0, .25),
   black-10: rgba(0, 0, 0, .10),
 
+  black-100: #DFDFDF,
```

Comment:
these are odd to me but I am assuming they were provided via design and I am don't have context 😉

Reply:
they are not provided by design, @steph-kalacis @erickrico could you provide the black hex codes here? I attempted to color pick the current rgba's over the white background to translate to hex.